lib/evaluate/host.cc

```diff
@@ -35,6 +35,7 @@ void HostFloatingPointEnvironment::SetUpHostFloatingPointEnvironment(
     return;
   }
 #if __x86_64__
+  HasSubnormalFlushingHardwareControl_ = true;
   if (context.flushSubnormalsToZero()) {
     currentFenv_.__mxcsr |= 0x8000;  // result
```

Comment:
Are you sure it's not GLIBC or Linux specific? fenv_t could be defined differently on various *BSD systems.

Reply:
To be honest, I do not know. I checked Bionic X86_64 after I saw your ifdef for it and it is is also  defining the `__mxcsr` field. I do not want to start reviewing all possible libc. We used this in flang without any libc consideration. 
I guess when someone tries compiling f18 with a X86_64 libc that does not have this, it won't compile here and they can open an issue and we will see what to do then.